app/assets/v2/js/navbar.js

```diff
@@ -186,18 +192,13 @@ const makeMenu = (navbarEl) => {
   // remove isVisible transitions to reduce jank
   const resetVisibility = () => {
     // removing isVisible will reset transition to just opacity
-    if (backgroundEl)
-      backgroundEl.classList.remove('isVisible');
-    if (contentEl)
-      contentEl.classList.remove('isVisible');
-    if (caretEl)
-      caretEl.classList.remove('isVisible');
+    backgroundEl.classList.remove('isVisible');
```

Comment:
@thelostone-mc added these `if`s a few days ago to fix some of the Sentry bugs. Do we need to remove them? Or does this branch just need to be updated with those changes?

Reply:
This negates the need to make those checks -- they never fully resolved the issue, that function shouldn't be getting called so we don't need to check for the els there.

Reply:
Makes sense ! if we can prevent it from being called at the start, that means my changes would be redundant

Reply:
Sounds good. Thanks for the extra info!